### PR TITLE
[FIX] Fix errors when userStatus empty

### LINKED ISSUE #9635: Tempalte errors on logout
### Description:

There's a series of template errors on logout, mostly related to account box helpers. I think maybe some helpers to update user status are running after the user is removed, before the template is removed, so they fail getting the status.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.61.0 (and open.rocket.chat)
* Operating System: Mac OS X
* Deployment Method(snap/docker/tar/etc): N/A
* Number of Running Instances: 1
* DB Replicaset Oplog: N/A
* Node Version: N/A

### Steps to Reproduce:

1. Open Chrome Inspector
2. Change settings to preserve logs
2. Logout

### Expected behavior:

No Errors

### Actual behavior:

```
Exception in template helper: TypeError: Cannot read property 'getUiText' of undefined
    at e.noSubscriptionText (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:411:53819)
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:31341
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:17175
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:31974
    at Function.e._withTemplateInstanceFunc (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:37698)
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:31881
    at p.call (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:145:1913)
    at p.mustacheImpl (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:145:1185)
    at Object.p.mustache (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:145:1244)
    at f.View._render (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:411:30772)
539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:1

Exception in template helper: TypeError: Cannot read property 'charAt' of undefined
    at Object.myUserInfo (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:411:33729)
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:31341
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:17175
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:31974
    at Function.e._withTemplateInstanceFunc (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:37698)
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:31881
    at Object.p.call (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:145:1928)
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:411:1083
    at f.View.<anonymous> (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:145:2786)
    at https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:19015
539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:1

Exception from Tracker recompute function: undefined
539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:1 Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at f._DOMRange.t.detach (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:5773)
    at Function.t._remove (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:3379)
    at f._DOMRange.t.detach (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:5702)
    at Function.t._remove (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:3379)
    at f._DOMRange.t.detach (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:5702)
    at Function.t._remove (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:3379)
    at f._DOMRange.t.detach (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:5702)
    at Function.t._remove (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:3379)
    at f._DOMRange.t.detach (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:5702)
    at Function.t._remove (https://open.rocket.chat/539c21bfd0b8a570b65fc22dca1869f7cfd87ee8.js?meteor_js_resource=true:79:3379) undefined
```

### PR DESCRIPTION
**Pull request by lunaticmonk**
@RocketChat/core 

Fixes #9635

### CODE REVIEW

File: packages/rocketchat-ui-sidenav/client/accountBox.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ Template.accountBox.helpers({
 		return {
 			name: Session.get(`user_${ username }_name`) || username,
 			status: Session.get(`user_${ username }_status`),
-			visualStatus: t(userStatus.charAt(0).toUpperCase() + userStatus.slice(1)),
+			visualStatus: userStatus ? t(userStatus.charAt(0).toUpperCase() + userStatus.slice(1)) : 'Offline',
```
Comment (geekgonecrazy) [inline packages/rocketchat-ui-sidenav/client/accountBox.js]: Only thing I can see is hardcoding to Offline its not getting translated.

Comment (sumedh123) [reply packages/rocketchat-ui-sidenav/client/accountBox.js]: @geekgonecrazy you mean to say internationalization?

Comment (geekgonecrazy) [reply packages/rocketchat-ui-sidenav/client/accountBox.js]: Translation/internationalization/i18n, take your pick 😉 all are talking about the same thing 🙂

Comment (sumedh123) [reply packages/rocketchat-ui-sidenav/client/accountBox.js]: @geekgonecrazy Should I include text in all languages out there?

Comment (geekgonecrazy) [reply packages/rocketchat-ui-sidenav/client/accountBox.js]: Why not just `t('Offline')` ?

**Comment by sumedh123** [comment]
@geekgonecrazy I did the changes but the circleci test is failing. I saw the logs but not getting a total view of what is happening out there. May I know how can I fix that? Thanks.

**Comment by karlprieb** [comment]
@sumedh123 can you resolve this conflict? so we can merge your PR! :)

**Comment by sumedh123** [comment]
@karlprieb I rebased my branch with "develop". I don't know but the latest changes on develop didn't have the whole file itself? Wanted to know why was that file deleted?

**Comment by sumedh123** [comment]
@karlprieb Also yesterday a release happened. Just wanted to know how does the release work? I mean all the changes approved from develop go directly in next release? I saw UI changes too.

**Comment by sumedh123** [comment]
@karlprieb Done 👍

**Comment by sumedh123** [comment]
Seems this error does not exist now.

**Comment by engelgabriel** [comment]
As @lunaticmonk said, seems this error does not exist now.

**Pull request closed without merging**